<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-28T11:23:09 IST" name="Default suite" finished-at="2024-04-28T11:23:17 IST" duration-ms="7796">
    <groups>
    </groups>
    <test started-at="2024-04-28T11:23:09 IST" name="Default test" finished-at="2024-04-28T11:23:17 IST" duration-ms="7796">
      <class name="api.test.PetTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:api.test.PetTest@55141def]" started-at="2024-04-28T11:23:09 IST" name="setUp" finished-at="2024-04-28T11:23:10 IST" duration-ms="225" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test_postPet()[pri:1, instance:api.test.PetTest@55141def]" started-at="2024-04-28T11:23:10 IST" name="test_postPet" finished-at="2024-04-28T11:23:13 IST" duration-ms="2963" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_postPet -->
        <test-method signature="test_getPetById()[pri:2, instance:api.test.PetTest@55141def]" started-at="2024-04-28T11:23:13 IST" name="test_getPetById" finished-at="2024-04-28T11:23:14 IST" duration-ms="1599" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_getPetById -->
        <test-method signature="test_updatePetById()[pri:3, instance:api.test.PetTest@55141def]" started-at="2024-04-28T11:23:14 IST" name="test_updatePetById" finished-at="2024-04-28T11:23:16 IST" duration-ms="1535" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [405]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at api.test.PetTest.test_updatePetById(PetTest.java:100)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_updatePetById -->
        <test-method signature="test_deletePetByID()[pri:4, instance:api.test.PetTest@55141def]" started-at="2024-04-28T11:23:16 IST" name="test_deletePetByID" finished-at="2024-04-28T11:23:17 IST" duration-ms="1423" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_deletePetByID -->
      </class> <!-- api.test.PetTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
